bruno/Create Account.bru

```diff
@@ -12,8 +12,8 @@ post {
 
 body:json {
   {
-      "name": "Sanjay Ramaswamy",
-      "email": "sanjay@ramaswamy.net",
-      "password": "ramaswamy123"
+      "name": "Tom Hanks",
+      "email": "woody@ramaswamy.net",
+      "password": "youareatoy"
```

Comment:
Changes to Bruno files should normally not be committed if you are just changing the parameters' contents for testing but leaving the actual keys themselves stay the same.

This is just to reduce diff noise.